include/deal.II/base/aligned_vector.h

```diff
@@ -1559,24 +1559,24 @@ AlignedVector<T>::replicate_across_communicator(const MPI_Comm &   communicator,
   // the shmem root!) but also destroy the MPI_Win and the communicator. All of
   // that is encapsulated in the following call where the deleter makes copies
   // of the arguments in the lambda capture.
-  elements =
-    decltype(elements)(aligned_shmem_pointer,
-                       [is_shmem_root,
-                        array_size,
-                        aligned_shmem_pointer,
-                        shmem_group_communicator,
-                        shmem_window](T *) mutable {
-                         if (is_shmem_root)
-                           for (unsigned int i = 0; i < array_size; ++i)
-                             aligned_shmem_pointer[i].~T();
-
-                         int ierr;
-                         ierr = MPI_Win_free(&shmem_window);
-                         AssertThrowMPI(ierr);
-
-                         ierr = MPI_Comm_free(&shmem_group_communicator);
-                         AssertThrowMPI(ierr);
-                       });
+  elements = decltype(elements)(aligned_shmem_pointer,
```

Comment:
I'm unsure why it wants to indent differently here...

Reply:
We must have not run clang-format one last time.